walker_test.go

```diff
@@ -290,12 +290,20 @@ func TestConvert(t *testing.T) {
 		},
 	}
 
-	gotFile := wlkr.convert(path, nil) // ensuring there is no problems with nil file stats.
+	gotFile, err := wlkr.convert(path, nil) // ensuring there is no problems with nil file stats.
+	if err != nil {
+		t.Fatal("unexpected error:", err)
+	}
+
 	if wantFile.Path != gotFile.Path {
 		t.Errorf("convert() path = %q; want: %q", gotFile.Path, wantFile.Path)
 	}
 
-	gotFile = wlkr.convert(path, info)
+	gotFile, err = wlkr.convert(path, info)
+	if err != nil {
+		t.Fatal("unexpected error:", err)
```

Comment:
Also here, simply `t.Fatal(err)` or add context like `t.Fatalf("wlkr.convert: %v", err)`. A failing test is already unexpected on its own.

Reply:
```suggestion
		t.Fatal(err)
```